modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/snapshot/SnapshotPartitionsQuickVerifyHandler.java

```diff
@@ -71,6 +72,22 @@ public SnapshotPartitionsQuickVerifyHandler(GridCacheSharedContext<?, ?> cctx) {
         String name,
         Collection<SnapshotHandlerResult<Map<PartitionKeyV2, PartitionHashRecordV2>>> results
     ) throws IgniteCheckedException {
+        IgniteLogger log = cctx.logger(SnapshotPartitionsQuickVerifyHandler.class);
+
+        Exception err = null;
+
+        for (SnapshotHandlerResult<Map<PartitionKeyV2, PartitionHashRecordV2>> result : results) {
+            if (result.error() != null) {
+                if (err == null)
+                    err = result.error();
+
+                log.error("Verify error", result.error());
```

Comment:
Why we need extra logging here? `IgniteSnapshotManager#completeHandlersAsyncIfNeeded()` logs handler result exceptions as a warning.

Reply:
Also, we might break the cycle after finding first error.